internal/auth/manager.go

```diff
@@ -269,13 +265,26 @@ func (m *Manager) authenticateJWT(req *Request) error {
 		return fmt.Errorf("JWT not provided")
 	}
 
-	var cc customClaims
-	_, err = jwt.ParseWithClaims(v["jwt"][0], &cc, keyfunc)
+	token, err := jwt.Parse(v["jwt"][0], keyfunc)
 	if err != nil {
 		return err
 	}
 
-	if !matchesPermission(cc.MediaMTXPermissions, req) {
+	claims := token.Claims.(jwt.MapClaims)[m.JWTClaimKey].([]interface{})
+	MediaMTXPermissions := make([]conf.AuthInternalUserPermission, 0, len(claims))
+
+	for _, permission := range claims {
+		var MediaMTXPermission conf.AuthInternalUserPermission
+		MediaMTXPermission.Action = conf.AuthAction(permission.(map[string]interface{})["action"].(string))
```

Comment:
this is also a possible cause of panics since permission might not be a `(map[string]interface{})`